iceoryx_utils/platform/win/include/iceoryx_utils/platform/wait.hpp

```diff
@@ -19,4 +19,9 @@ inline int WEXITSTATUS(int)
     return 0;
 }
 
+inline int waitpid(int pid, int* stat_loc, int options)
```

Comment:
what is the consequence of this default implementation?

Reply:
Where do we use it?

Reply:
I don't think it's used in iceoryx. The only matches I get are within googletest, which is obviously not including this header.